src/Console/ModelsCommand.php

```diff
@@ -458,7 +462,7 @@ protected function getPropertiesFromMethods($model)
                 } elseif (Str::startsWith($method, 'set') && Str::endsWith(
                     $method,
                     'Attribute'
-                ) && $method !== 'setAttribute'
+                ) && $method !== 'setAttribute' && $method !== 'setClassCastableAttribute'
```

Comment:
I just realized why this exclusion of this method is in :-/

Reply:
This is the [standard Laravel](https://github.com/laravel/framework/blob/7.x/src/Illuminate/Database/Eloquent/Concerns/HasAttributes.php#L725-L751) method in `protected` visibility required for custom attribute handling.

I doubt that it needs to be added to the helper and even in the form of `class_castable`.

Reply:
If necessary, I can remove the exception of this method from the code.
But this is the standard [Laravel](https://github.com/laravel/framework/blob/7.x/src/Illuminate/Database/Eloquent/Concerns/HasAttributes.php#L725-L751) method in protected visibility required for custom attribute handling.

Reply:
I think it's fine and makes sense.

Only other way to "fix" this would be to change the name in Laravel.

Reply:
I doubt that Taylor will agree to this)

Reply:
Me too 😄